mne_features/tests/test_univariate.py

```diff
@@ -254,6 +255,61 @@ def test_energy_freq_bands():
     assert_equal(band_energy > 0.98 * tot_energy, True)
 
 
+def test_powercurve_deviation():
+    """Test for :func:`compute_powercurve_deviation`.
+
+    We impose the power to be written as power(f) = k1/f**theta with noise
+    and derive a signal candidate, to which we apply the function
+    compute_powercurve_deviation and check whether the k1 and theta estimates
+    are correct.
+    """
+    sfreq = 2034.51
+
+    # Support of the spectrum
+    freqs = np.fft.fftfreq(n=int(sfreq), d=1. / sfreq)
+
+    # parameters
+    k1 = 5.
+    theta = 3.
+
+    # Define the magnitude of the spectrum
+    # such that power(f) = k1/f**a with noise
+    mag = np.zeros((freqs.shape[0],))
+    mag[0] = 0
+    for f in range(1, 128):
+        m = np.random.uniform(low=-0.01, high=0.01)
+        mag[f] = (np.sqrt(k1) + m) / (f ** (theta / 2))
+        mag[-f] = mag[f]
+
+    # From the magnitude, we get the spectrum, choosing a random phase per bin.
+    spect = np.zeros((freqs.shape[0],), dtype=np.complex64)
+    spect[0] = 0
+    for f in range(1, 128):
+        phase = np.random.uniform(low=-np.pi, high=np.pi)
+        spect[f] = mag[f] * np.exp(phase * 1j)
+        spect[-f] = np.conj(spect[f])
+
+    # Take the inverse FFT to go back to the time domain.
+    # The imaginary part of _sig is numerically close to 0 (hermitian symmetry).
+    # So we obtain a real signal
+    _sig = np.fft.ifft(spect)
+    sig = _sig.real
+    n_times = sig.shape[0]
+
+    # We test our estimates
+    data = sig.reshape(1, -1)
+    intercept, slope, mse, r2 = compute_powercurve_deviation(sfreq=sfreq, data=data, with_intercept=True)
```

Comment:
This line is probably too long. Check it with `Flake8` before you commit unless Travis will complain...